modules/odc-adding-services-to-application.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// applications/application_life_cycle_management/odc-viewing-application-composition-using-topology-view.adoc
+
+[id="odc-adding-services-to-your-application_{context}"]
+= Adding services to your application
+
+In the *Topology view*, access the *+Add* actions using the context menu, sidebar, or by hovering and dragging the dangling arrow on to the *Topology view* to add a service to your application.
```

Comment:
```suggestion
To add a service to your application, in the *Topology view*, access the *+Add* actions using the context menu, sidebar, or by hovering and dragging the dangling arrow on to the *Topology view*.
```

Reply:
Topology graph?

Reply:
Is it better to keep this abstract lean and not mention the different UI methods here? (`context menu, sidebar, or by hovering and dragging the dangling arrow` ) 

The procedure focuses on one specific UI method, the context menu. If the other methods are important for users to know about, you could mention them in a note. For example,
```
[NOTE]
====
In addition to the context menu, you can add services by using the sidebar or hovering and dragging the dangling arrow to the Topology graph
====
```